Cargo.toml

```diff
@@ -48,6 +48,9 @@ edition = "2021"
 homepage = "https://astar.network"
 repository = "https://github.com/AstarNetwork/Astar"
 
+[replace]
```

Comment:
`patch` is recommended as replace has been deprecated for a long time now
https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#the-replace-section

OR

Just add it deps to force use specific patch release

```
yamux = "0.13.4"
```


Lastly, let's add a TODO to remove this once the linked issue is fixed

Reply:
Thanks TIL